frontend/src/routes/playground/+page.svelte

```diff
@@ -42,9 +43,17 @@
 </div>
 
 <div class="flex h-full w-full flex-col p-4">
-	<h1 class="mb-2 text-2xl font-bold">Zeno View Playground</h1>
+	<div class="mb-4 flex h-12 items-center">
+		<a href="/" class="shrink-0">
+			<img src="/zeno-logo.png" class="h-8" alt="diamond tesselation logo" />
+		</a>
+		<h1 class="ml-3 text-2xl font-bold">Instance View Playground</h1>
+		<div class="ml-auto mr-2">
+			<Button variant="raised" href="https://zenoml.com/docs/views/">Read the Docs</Button>
+		</div>
+	</div>
 	<div class="flex w-full items-center pb-4">
-		<h2 class="pr-4 text-xl">Examples:</h2>
+		<h2 class="pr-4 text-xl">Standard Views:</h2>
```

Comment:
Separate PR? Fine for now but generally would.

Reply:
Yeah probably should have been